src/owui_ext/shared/parsing.py

```
@@ -0,0 +1,35 @@
+
+    LLM responses often surround the JSON object with prose ("Here's the
+    answer: {..}"). The fallback grabs the slice between the first ``{`` and
+    the last ``}`` and retries; if that also fails we give up.
+    """
+    if not text:
+        return None
+    text = text.strip()
+    try:
+        return json.loads(text)
+    except Exception:
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)